server/views.py

```diff
@@ -40,9 +61,18 @@ def home():
 def create_auth_token():
     # get server token from header (convert to str to fix weird encoding issue on production)
     server_token = request.headers.get('X-Server-Token', type=str)
+    user = str(request.get_json()['user'])
+    name = str(request.get_json()['name'])
+    email = str(request.get_json()['email'])
+    duration = int(request.get_json()['duration'])
```

Comment:
Store `request.get_json()` in a local variable instead of repeatedly re-requesting.